src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.GenerateAssemblyInfo.targets

```diff
@@ -123,6 +123,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     
     <PropertyGroup>
       <FileVersion Condition="'$(FileVersion)' == ''">$(AssemblyVersion)</FileVersion>
+      <InformationalVersion Condition="'$(InformationalVersion)' == ''">$(AssemblyVersion)</InformationalVersion>
```

Comment:
Ok, I wasn't sure which would be better. I don't remember my reasoning for going this way so I'll switch to what you agree on.